cli/flags.rs

```diff
@@ -730,6 +731,17 @@ Generate html reports from lcov:
         .help("Output coverage report in lcov format")
         .takes_value(false),
     )
+    .arg(Arg::new("out_file")
```

Comment:
Rename to `output`, it's unwieldy to require underscores in CLI flags

Reply:
there were couple of `out_file` arg name used for same logic, and some how I feel like `--out-file` is kinda more descriptive than `--output` ?
https://github.com/denoland/deno/blob/380ebbd6d55f8933f614c405ee1050f7231d7ed4/cli/flags.rs#L559-L562

Reply:
anyways, if you want it to be `--output` please let me know again, I'll change it !!

Reply:
`out_file` is only used in `deno bundle` and it is a positional argument so you don't type out it's name explicitly. Let's go with `output` which matches `deno compile`.